<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="1" name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="1" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="sokoeurnchhayacademy.test.SubmitOrderTest">
        <methods>
          <include name="orderHistoryTest"/>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="submitOrder" invocation-numbers="0 1"/>
        </methods>
      </class> <!-- sokoeurnchhayacademy.test.SubmitOrderTest -->
      <class name="sokoeurnchhayacademy.test.ErrorValidationTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="ProductErrorValidation"/>
        </methods>
      </class> <!-- sokoeurnchhayacademy.test.ErrorValidationTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
